test/e2e/ansible/cluster_test.go

```diff
@@ -60,7 +60,7 @@ var _ = Describe("Running ansible projects", func() {
 			Expect(metrics.CleanUpMetrics(kctl, metricsClusterRoleBindingName)).To(Succeed())
 
 			By("cleaning up created API objects during test process")
-			Expect(operator.UndeployOperator(ansibleSample)).To(Succeed())
+			testutils.WrapWarnOutput("", operator.UndeployOperator(ansibleSample))
```

Comment:
Is Undeploy causing some issues? Currently, there is only one `It`, so it's _okay_ to use `Warn` instead of `Error`, but if the test grows, it could be an issue, and  I think upstream won't accept this change.

Reply:
The `make undeploy` removes the objects in the same order they are created. The namespace is deleted first and as it gets deleted all the resources created in the namespaces also gets deleted along with it. This always fails when running in an OCP cluster. However, on a kind cluster, this doesn't fail as the deletion of namespace takes some time and thus the `make undeploy` completes successfully.

I think it's okay to not fail a test if the cleanup fails. It's not related to the actual e2e test anyway.